jxmlease/_basenode.py

```diff
@@ -96,7 +96,7 @@ def __init__(self):
         the node.
     tag (string): The XML tag for this node.
     key (string or tuple): The dictionary key used for this node.
-    xml_attrs (dict): The XML attributes for the node.
+    xml_attrs (dictionary): The XML attributes for the node.
```

Comment:
Agreed. In testing, I tried multiple ways of escaping these... but, not this way. ;-)

I'll change them.